src/erasure-code/jerasure/jerasure_init.cc

```diff
@@ -18,13 +18,15 @@
 #include "common/debug.h"
 #include "jerasure_init.h"
 
+#ifdef __cplusplus
```

Comment:
this is not necessary. .cc file is compiled as C++ source file by CMake.

Reply:
I compile it on ubuntu 22, and find vstart.sh will stop after print "load dlopen(ceph/build/lib/libec_jerasure.so): /ceph/build/lib/libec_jerasure.so: undefined symbol: jerasure_init". test this pr can fix it
is it ok?

Reply:
> is it ok?

it's okay. but it's not enough. you need to convince the reviewer by reasoning not by observations.

Reply:
I runned nm, "nm lib/libec_jerasure.so | grep -w jerasure_init" print:
"U jerasure_init"

but i can't see it again after update and recompile, not very sure about it, just close it and try repetition latter